src/Draco.Debugger/Debugger.cs

```diff
@@ -98,6 +98,11 @@ public sealed class Debugger
 
     private readonly SessionCache sessionCache = new();
 
+    /// <summary>
+    /// Unused, but signal ownership, help debugging...
+    /// </summary>
+    private readonly Task ioWorkerTask;
```

Comment:
Maybe will be threw away depending on how we refactor further ioWorker, but if we keep fire&forget the task from the constructor, we can at least keep a reference to the task.
It's better for debugging purpose if the exception throw in background, but also signal the ownership.